src/cryptonote_basic/cryptonote_format_utils.h

```diff
@@ -89,13 +89,16 @@ namespace cryptonote
   void set_encrypted_payment_id_to_tx_extra_nonce(blobdata& extra_nonce, const crypto::hash8& payment_id);
   bool get_payment_id_from_tx_extra_nonce(const blobdata& extra_nonce, crypto::hash& payment_id);
   bool get_encrypted_payment_id_from_tx_extra_nonce(const blobdata& extra_nonce, crypto::hash8& payment_id);
-  bool is_out_to_acc(const account_keys& acc, const txout_to_key& out_key, const crypto::public_key& tx_pub_key, const std::vector<crypto::public_key>& additional_tx_public_keys, size_t output_index);
+  void set_tx_out(const uint64_t amount, const crypto::public_key output_public_key, const bool use_view_tags, const crypto::view_tag view_tag, tx_out &out);
```

Comment:
I am suddenly not sure myself anymore, but isn't it custom to pass almost **any** structured and/or somewhat larger things with `const &` instead of only `const`, like e.g. here `output_public_key`?

Reply:
You're right I'll update to `const &`